compute_tools/src/bin/compute_ctl.rs

```diff
@@ -30,7 +30,8 @@
 //!             -C 'postgresql://cloud_admin@localhost/postgres' \
 //!             -S /var/db/postgres/specs/current.json \
 //!             -b /usr/local/bin/postgres \
-//!             -r {"bucket": "my-bucket", "region": "eu-central-1", "endpoint": "http:://localhost:9000"} \
+//!             -r {"bucket": "my-bucket", "region": "eu-central-1", "endpoint": "http:://localhost:9000",
+//!             (optionally) "key": "AWS_SECRET_ACCESS_KEY", "id": "AWS_ACCESS_KEY_ID"}
```

Comment:
why are these fields optional? what are defaults?

In my understanding full `-r` remote_config may be optional, but not these fields.

Reply:
they are optional because you might already have environment variables configured. 
if so there is no need to re-specify the key / id